src/components/NewWorkspaceModal.test.ts

```diff
@@ -358,26 +346,21 @@ describe('NewWorkspaceModal', () => {
     // Arrange
     const user = userEvent.setup();
 
-    Ajax.mockImplementation(() => ({
-      Billing: {
-        listProjects: async () => [gcpBillingProject, azureBillingProject],
-      },
-      ...hasGroupsAjax,
-    }));
+    asMockedFn(Ajax).mockImplementation(
+      () =>
+        ({
+          Billing: {
+            listProjects: async () => [gcpBillingProject, azureBillingProject],
+          },
+          ...hasGroupsAjax,
+        } as AjaxContract)
+    );
 
     await act(async () => {
-      // eslint-disable-line require-await
       render(
         h(NewWorkspaceModal, {
-          cloneWorkspace: false,
-          onSuccess: () => null,
-          onDismiss: () => null,
-          customMessage: null,
-          requiredAuthDomain: false,
-          requireEnhancedBucketLogging: true,
```

Comment:
This test is supposed to be testing if secure bucket logging is enabled when an auth domain is selected, but it had `requireEnhancedBucketLogging: true`, which makes secure bucket logging always enabled.